admin_scripts/systemd_policy_cleanup.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+#================================================================
+# HEADER
+#================================================================
+#% SYNOPSIS
+#+    systemd_policy_cleanup.sh [ENFORCE]
+#%
+#% DESCRIPTION
+#%    This script installs a systemd unit that resets the contents of "user"'s
+#%    home directory at system startup and shutdown time.
+#%
+#%    It takes one optional parameter: whether or not to enforce this policy.
+#%    If this parameter is missing, empty, "false", or "falsk", the policy will
+#%    be removed; otherwise, it will be enforced.
+#%
+#================================================================
+#- IMPLEMENTATION
+#-    version         systemd_policy_cleanup.sh (magenta.dk) 1.0.0
+#-    author          Alexander Faithfull
+#-    copyright       Copyright 2019, 2020 Magenta ApS
+#-    license         GNU General Public License
```

Comment:
When puttiing something under the GPL we should state which version. I think "3 or later" must be the right choice.

Reply:
Updated to `GNU General Public License, version 3 or later`.